src/coreclr/vm/methodtablebuilder.cpp

```diff
@@ -2943,10 +2946,6 @@ MethodTableBuilder::EnumerateClassMethods()
                 {
                     BuildMethodTableThrowException(BFA_VIRTUAL_PINVOKE_METHOD);
                 }
-                if(IsMdStatic(dwMemberAttrs))
```

Comment:
We should not permit static virtual methods on classes/structs. By our ecma design, those should continue to throw.